### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Yilikal250**
- [ ] Created a table named owners with columns:
- [ ] Created a table named species with columns:
- [ ] Modified animals table:
- [ ] Inserted data into the owner's table:
- [ ] Inserted data into the species table:
- [ ] Modified and inserted animals so it includes the species value:
- [ ] Modified your inserted animals to include owner information (owner_id):
- [ ] Wrote queries (using JOIN) to answer the questions:
- [ ] 3rd review
- [ ] 3rd 2

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Yilikal250, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has old README file. Please update the new one, here is the [Link](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏a

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by roshan-bajgain** [comment]
Hello @Yilikal250, you were right please use the same one you used before. Sorry for the misunderstanding. README file for vet-clinic, [Link](https://github.com/microverseinc/curriculum-template-databases)

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,22 @@ VALUES
...
+('Ditto', DATE '2022-05-14', 4, 'TRUE', 22);
+
+INSERT INTO owners (full_name, age)
+VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
```
Comment (DeliceLydia) [inline data.sql]: - [ ] Add data in the `species` table
```sql
INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Yilikal250, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: data.sql (hunk 1 above)
Comment (rloterh) [reply data.sql]: well resolved! ✔️

File: README.md [hunk 2]
```diff
@@ -1,45 +1,203 @@
...
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
 
-This project is [MIT](./MIT.md) licensed.
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (rloterh) [inline README.md:201]: - [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove guide note below it.

**Review by rloterh — COMMENTED** [review]
Checked!

File: README.md (hunk 2 above)
Comment (rloterh) [inline README.md:201]: - [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove guide note below it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Yilikal,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉💯🌟 

## Highlights 👍
- Queries properly implemented

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**